pkg/flowaggregator/flowaggregator.go

```diff
@@ -398,6 +412,17 @@ func (fa *flowAggregator) flowRecordExpiryCheck(stopCh <-chan struct{}) {
 					continue
 				}
 			}
+			// Initialize sarama Kafka producer
+			if fa.kafkaProducer != nil {
+				if fa.kafkaProducer.GetSaramaProducer() == nil {
+					err := fa.kafkaProducer.InitSaramaProducer()
+					if err != nil {
+						klog.Errorf("Error when initializing kafka producer: %v, will retry in %s", err, fa.activeFlowRecordTimeout)
+					} else {
+						klog.Info("Initialized kafka producer")
```

Comment:
please use structured logging for all new log messages

Reply:
done